Configuration/PyReleaseValidation/python/relval_2017.py

```diff
@@ -47,8 +48,8 @@
 #        (TTbar DigiNoHLT)
 #   2023 (TTbar, TTbar PU, TTbar PU premix)
 #        (TTbar trackingMkFit)
-#        (TTbar FastSim)
-#   2024 (TTbar, TTbar PU, TTbar PU premix)
+#        (TTbar FastSim, Nu Gun)
```

Comment:
It seems the text and workflows are in different order. Should it be 

```
(TTbar FastSim)
(Nu Gun)
2024 ....
(Nu Gun)
```
and the workflow order 
```
14034.0,
12461.0,
12834.0,13034.0,13034.99,
12861.0,
```

Reply:
This has not been resolved, please fix.

Reply:
Fixed, took the chance to add extra comments in `upgradeWorkflowComponents.py`.